### PR TITLE
Add refresh button

### PR DESCRIPTION
**Pull request by inquisitiv-e**
- Added refresh button 
- Users can see if there are any new postings made by other users since the website was loaded.

Tested by deleting an item manually in the db and press the button
Screen Shot 2019-07-06 at 11 58 04 AM

### CODE REVIEW

**Comment by YaoLiu93** [comment]
I wonder if adding pub/sub will remove the requirement for this... but I'll look into that when I get to it.

**Comment by YaoLiu93** [comment]
Can you include a screen shot of the button? It doesn't appear to be styled?

**Comment by YaoLiu93** [comment]
Can you merge master in, and then style the button to fit? :D

**Comment by nicolehli** [comment]
> I wonder if adding pub/sub will remove the requirement for this... but I'll look into that when I get to it.

I'm going to hold off on approving this. No specific issues, but since the button is not in our requirement and Yao's comment about pub/sub. We can hold off on merging this and review this together on Wednesday when Yao is back.

**Comment by nicolehli** [comment]
@inquisitiv-e we want to look at this tonight, can you resolve conflicts?

**Comment by inquisitiv-e** [comment]
> @inquisitiv-e we want to look at this tonight, can you resolve conflicts?

RESOLVED

File: imports/ui/components/RefreshButton.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import Button from "@material-ui/core/Button";
+import Renew from "@material-ui/icons/Autorenew";
+import "./RefreshButtonStyle.css"
+
+class RefreshButton extends Component {
```
Comment (YaoLiu93) [inline imports/ui/components/RefreshButton.jsx]: This probably doesn't need to be a class, but that's fine.

Comment (inquisitiv-e) [reply imports/ui/components/RefreshButton.jsx]: Okayyyy

**Pull request merged**